storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -36,8 +36,8 @@ include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
 
 include::modules/persistent-storage-csi-vsphere-stor-policy.adoc[leveloffset=+1]
 
-[id="vsphere-csi-driver-requirements_{context}"]
-include::snippets/vmware-csi-driver-reqs.adoc[leveloffset=+1]
+include::modules/vmware-csi-driver-reqs.adoc[leveloffset=+1]
+For more information about removing a third-party CSI driver, see xref:../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-install-issues_persistent-storage-csi-vsphere[Removing a third-party vSphere CSI Operator Driver].
```

Comment:
Same here - this needs to be in an additional resources section.

Reply:
Fixed